app-benchmarks/sysbench/sysbench-1.0_pre20161017.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+GITHUB_REV="48124f838b00ff83a044fbf046a9d8d0b1602d90"
+MY_PN="${PN}-${GITHUB_REV}"
+
+DESCRIPTION="System performance benchmark"
+HOMEPAGE="https://github.com/akopytov/sysbench"
+
+SRC_URI="https://github.com/akopytov/sysbench/archive/${GITHUB_REV}.zip -> ${MY_PN}.zip"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="aio lua mysql postgres test"
+
+CDEPEND="aio? ( dev-libs/libaio )
+	lua? ( dev-lang/lua:= )
+	mysql? ( virtual/libmysqlclient )
+	postgres? ( dev-db/postgresql:= )"
+
+DEPEND="${CDEPEND}
+	sys-devel/libtool:=
+	dev-libs/libxslt
+	test? ( dev-util/cram )"
+RDEPEND="${CDEPEND}"
+
+REQUIRED_USE="
+	mysql? ( lua )
+	postgres? ( lua )"
```

Comment:
The database tests are written in Lua (https://github.com/akopytov/sysbench/tree/1.0/sysbench/tests/db) and without Lua support you cannot run them.